app/views/macros/search-form.njk

```diff
@@ -6,6 +6,7 @@
     {% for input in model.hiddenInputs %}
       <input type="hidden" id="{{input.id}}" name="{{input.name}}" value="{{input.value}}"/>
     {% endfor %}
+    <input type="hidden" id="sortBy" name="sortBy" value="score"/>
```

Comment:
Please add this hidden input via the model. If you look at 3 lines above this line, the model looks at the hiddenInputs fiels which is a list of objects. We are already passing pageId from the results.ts so if you just add another object for sortBy it will automatically be picked up here. Other reason is that search-form is also used on the search page and sortBy is not needed on that page. With the hiddenInputs, we can control when to add this field.

Reply:
agreed, done. make uniform.